community-containers/borgbackup-viewer/readme.md

```diff
@@ -0,0 +1,15 @@
+## Borgbackup Viewer
+This container allows to view the local borg repository in a web session. It also allows you to restore files and folders from the backup by using desktop programs in a web browser.
+
+### Notes
+- After adding and starting the container, you need to visit `https://ip.address.of.this.server:5800` in order to log in with the user `nextcloud` and the password that you can retrieve when running `sudo docker inspect nextcloud-aio-borgbackup-viewer | grep WEB_AUTHENTICATION_PASSWORD`.
+- Then, you should see a terminal. There type in `borg mount /mnt/borgbackup/borg /tmp/borg` to mount the backup archive. You need to type in the password for borg next that is shown in the AIO interface. Afterwards type in `nautilus /tmp/borg` which will show a file explorer and allows you to see all the files. You can then copy files and folders back to ...
```

Comment:
TODO: export the password automatically as even typing in the command is complicated enough.

Reply:
also write to use ctrl + c and to type in umount /tmp/borg after the person is done